arangosh/Dump/DumpFeature.cpp

```diff
@@ -750,6 +750,11 @@ Result DumpFeature::runDump(httpclient::SimpleHttpClient& client, std::string co
     _options.tickEnd = tick;
   }
 
+  VPackSlice const properties = body.get(StaticStrings::Properties);
+  if (!properties.isObject()) {
+    return ::ErrorMalformedJsonResponse;
```

Comment:
I think we should **not** make things fail here.
Otherwise it will not be possible to create a dump of a 3.5 database with a devel arangodump.
Please handle a missing "properties" attribute gracefully.